framework/src/image_data.cpp

```diff
@@ -2,7 +2,8 @@
 
 namespace gvk
 {	
-	// implementation in bridge pattern
+	/** Concrete class that represents cubemap image data composed from individual image files
```

Comment:
I would rather describe it as "concrete implementation", not as "concrete class". But maybe some more details could be added.

Reply:
I am using the technical term "concrete class" from OOP in the sense that this class can be instantiated, as opposed to its base class which is abstract. 
However, Bjarne Stroustrup seems to use a more refined notion of concrete types and concrete classes, cf. e.g. this discussion: https://softwareengineering.stackexchange.com/questions/221437/concrete-types-as-described-by-stroustrup-c-programming-language-4th-ed
In the context of C++ the class is probably best described as an implementation of the image_data_implementor interface. The C++ Programming Language also mentions the concept of node classes as a form of implementation inheritance in class hierarchies. This class also falls in the definition of concrete node classes.